README.md

```diff
@@ -766,3 +770,19 @@ or, optionally, you can override the default timezone:
 ```sql
 {{ dbt_date.yesterday(tz="America/New_York") }} as date_yesterday
 ```
+
+### [relative_datediff](macros/utils/relative_datediff.sql)(`date1, date2, datepart`)
+
+A date diff macro that can account for non calendar computed diffs between two date like fields, `date1` and `date2`, measured in `datepart`.
+
+When `day` is passed as the `datepart`, this reduces to the usual dbt utils datediff.
+
+For example,
+`{{dbt_date.relative_datediff("'2021-11-05'", "'2021-12-02'", 'month')}}` returns 0 while
```

Comment:
Couple of small nits (1):
`{{dbt_date.` should be
`{{ dbt_date.` (add space b/w `{{` and `dbt_date`). 
Same for next line.